piwik/content.md

```diff
@@ -0,0 +1,71 @@
+# Piwik
+
+[![Build Status](https://travis-ci.org/piwik/docker-piwik.svg?branch=master)](https://travis-ci.org/piwik/docker-piwik)
+
+Piwik is the leading open-source analytics platform that gives you more than just powerful analytics:
+ - Free open-source software
+ - 100% data ownership
+ - User privacy protection
+ - User-centric insights
+ - Customisable and extensible
+
+%%LOGO%%
+
+# How to use this image
+
+The easiest is to use our `docker-compose.yml`.
+
+Make sure you have [docker-compose](http://docs.docker.com/compose/install/) installed. And then:
+
+```
+git clone https://github.com/piwik/docker-piwik.git
+cd docker-piwik
+# edit variables:
+vi .env
+docker-compose up
+```
+
+You can now access your instance on the port 80 of the IP of your machine.
```

Comment:
I think it is worth mentioning that this is good for a test environment, but not a secure setup for public-facing instances.